packages/opentelemetry-plugin-postgres/src/pg.ts

```diff
@@ -84,27 +75,45 @@ export class PostgresPlugin extends BasePlugin<typeof pgTypes> {
         // Handle different client.query(...) signatures
         if (typeof args[0] === 'string') {
           if (args.length > 1 && args[1] instanceof Array) {
-            _handleParameterizedQuery.call(this, span, ...args);
+            utils._handleParameterizedQuery.call(this, span, ...args);
```

Comment:
If this function is moving to another module, then should the `_` prefix be removed?

Reply:
Agreed. Removed the prefixes